### PR TITLE
fix #64745 loveheaven.net

### LINKED ISSUE #64745: loveheaven.net
### Issue URL (Ads)
 [https://loveheaven.net/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Floveheaven.net%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.0.3
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | enabled
AdGuard DNS: | Default
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese
Other extensions used: | 280blocker

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/64745

### CODE REVIEW

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||jiltlargosirk.com^
```
Comment (Alex-302) [inline EnglishFilter/sections/adservers.txt:8]: is not it blocked by `/^(?!.*(google\.com|gstatic\.com|googleapis\.com|recaptcha\.net|cdn\.ampproject\.org|fluidplayer\.com|facebook\.net|fbcdn\.net|cloudflare\.com|ebacdn\.com|vk\.com|jquery\.com|jsdelivr\.net|yastatic\.net|yandex\.net|ytimg\.com|fontawesome\.com|dmca\.com|disqus\.com|twitter\.com|vidazoo\.com|disquscdn\.com|jwpcdn\.com|shrink\.pe|zencdn\.net|unpkg\.com|github\.io|ulogin\.ru|bootstrapcdn\.com|player)).*$/$third-party,script,domain=news.techrfour.com|shortzzy.in|shrinkme.in|shorten.sh|gibit.xyz|btcdot.xyz|iir.ai|shrinkbtc.cc|todaynewspk.win|cut.my.id|ouofly.com|stfly.me|dz4win.com|bolssc.com|genpas.icu|canyoublockit.com|aii.sh|uptoshort.com|gsurl.in|oncehelp.com|adslinkfly.online|loveheaven.net|extramovies.casa|asianclub.nl|file-up.org|kshow123.net|short.articlix.com|dlapk4all.com|cryptoads.space|profitlink.info|shorthitz.com|putlocker.digital|pimpandhost.com|pornpapa.com|shorthub.co|xxxfiles.com|pornone.com|rawdex.net|hiperdex.com|ask4movie.co|scash.xyz|animekayo.com|animekaizoku.com|xorgasmo.com|mitly.us|xtits.com|vjav.com|japscan.se` form Base filter?

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/adservers.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/adservers.txt:8]: in any case domain rule will be useful

Comment (Yuki2718) [reply EnglishFilter/sections/adservers.txt:8]: IDK, does regex work on iOS? I get the domain from Activity log.

Comment (Yuki2718) [reply EnglishFilter/sections/adservers.txt:8]: Tested without Simplified filter option, still get the popup and jiltlargosirk unblocked. So apparently the rule is not working on my iOS 12.4.8 + AG for iOS 4.0.3(544).

**Pull request merged**